src/main/resources/application.properties

```diff
@@ -39,6 +39,8 @@ cryostat.http.proxy.host=${quarkus.http.host}
 cryostat.http.proxy.port=${quarkus.http.port}
 cryostat.http.proxy.path=/
 
+cryostat.target.uri-range=SITE_LOCAL
```

Comment:
I think this can default at least to `DNS_LOCAL` if not even to `PUBLIC`. It makes sense to offer as an additional security control for administrators, but we don't want the defaults to be so restrictive that it just confuses or frustrates users who aren't concerned about this potential security aspect.